op-mode-definitions/show-configuration.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+    <node name="configuration">
+      <properties>
+        <help>Show available saved configurations</help>
+      </properties>
+      <!-- no admin check -->
+      <command>cli-shell-api showCfg --show-active-only --show-hide-secrets</command>
+      
+      <children>
+        <node name="all">
+          <properties>
+            <help>Show running configuration (including default values)</help>
+          </properties>
+          <!-- no admin check -->
+          <command>cli-shell-api showCfg --show-show-defaults --show-active-only --show-hide-secrets</command>
+        </node>
+        <node name="commands">
+          <properties>
+            <help> Show running configuration as set commands </help>
+          </properties>
+          <!-- no admin check -->
+          <command>${vyatta_sbindir}/vyatta-config-gen-sets.pl</command>
+        </node>
+        <node name="files">
+          <properties>
+            <help> Show available saved configurations </help>
+          </properties>
+          <!-- no admin check -->
+          <command>${vyos_op_scripts_dir}/show-configuration-files.sh</command>
```

Comment:
I don't see this scripts. I'm missing something, or you forgot to commit it?

Reply:
File commited.. i forgot to commit it...